internal/ingress/controller/nginx.go

```diff
@@ -784,6 +816,48 @@ func configureDynamically(pcfg *ingress.Configuration, port int) error {
 	return nil
 }
 
+// configureCerts JSON encodes certificates and POSTs it to an internal HTTP endpoint
+// that is handled by Lua
+func configureCerts(pcfg *ingress.Configuration, port int) error {
+	var servers []*ingress.Server
+
+	for _, server := range pcfg.Servers {
+		servers = append(servers, &ingress.Server{
+			Hostname: server.Hostname,
+			SSLCert: ingress.SSLCert{
+				PemCertKey: server.SSLCert.PemCertKey,
+			},
+		})
+	}
+
+	buf, err := json.Marshal(servers)
+
```

Comment:
nitpick, let's get rid of the new lines between err assignment and returning. Also have you run `gofmt`? 
```
_, err = ...
if err != nil {
   return err
}
```